index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>ADOPTADOG</title>
```

Comment:
one thing that's helpful for title elements is to use a `\` character to seperate the site name from the specific page name, it's a nice touchl

`<title>Adoptadog | Contact</title>`
`<title>Adoptadog | Home</title>`
etc